packages/auth/src/semantics.ts

```diff
@@ -7,16 +7,14 @@ Resource name: 'adx'
 
 - Full permission for account: 
     adx://did:example:userDid/*
-- Permission to write to particular application namespace: 
-    adx://did:example:userDid/namespace/*
-- Permission to write objects within a particular collection (namespace + dataset): 
-    adx://did:example:userDid/namespace/post/*
+- Permission to write to particular application collection: 
+    adx://did:example:userDid/com.foo.post/*
 - Permission to create a single interaction on user's behalf: 
-    adx://did:example:userDid/namespace/post/234567abcdefg
+    adx://did:example:userDid/com.foo.post/234567abcdefg
```

Comment:
The change in semantics here means that we can't delegate a UCAN to for `com.foo.*`

this is probably okay for now, I can come back and fix this up if we decide to stick with UCANs for auth

Reply:
That occurred to me while I was making the changes. I decided not to do anything yet, but I think we can definitely add the `*` semantic in the future.